codecov.yaml

```diff
@@ -7,6 +7,32 @@ coverage:
         informational: true # We don't want to block PRs based on the result
     project:
       default:
-        target: 90
+        target: 50
         threshold: 5
-        informational: true
+        informational: true # We don't want to block PRs based on the result
+
+flags:
```

Comment:
This will ensure that we don't override coverage of unrelated subfolders, i.e. when router pr is merged, cli coverage will still be maintained.